stubs/lief/lief/DEX.pyi

```diff
@@ -0,0 +1,416 @@
+    @property
+    def has_class(self) -> bool: ...
+    @property
+    def index(self) -> int: ...
+    @property
+    def is_virtual(self) -> bool: ...
+    @property
+    def name(self) -> str: ...
+    @property
+    def prototype(self) -> Any: ...
+
+class Prototype(lief.Object):
+    def __init__(self, *args, **kwargs) -> None: ...
+    def __eq__(self, arg0: Prototype) -> bool: ...
+    def __hash__(self) -> int: ...
+    def __ne__(self, arg0: Prototype) -> bool: ...
+    @property
+    def parameters_type(self) -> lief.DEX.Prototype.it_params: ...
+    @property
+    def return_type(self) -> Any: ...
+
+class Type(lief.Object):
+    class PRIMITIVES:
+        __members__: ClassVar[dict] = ...  # read-only
+        BOOLEAN: ClassVar[Type.PRIMITIVES] = ...
+        BYTE: ClassVar[Type.PRIMITIVES] = ...
+        CHAR: ClassVar[Type.PRIMITIVES] = ...
+        DOUBLE: ClassVar[Type.PRIMITIVES] = ...
+        FLOAT: ClassVar[Type.PRIMITIVES] = ...
+        INT: ClassVar[Type.PRIMITIVES] = ...
+        LONG: ClassVar[Type.PRIMITIVES] = ...
+        SHORT: ClassVar[Type.PRIMITIVES] = ...
+        VOID_T: ClassVar[Type.PRIMITIVES] = ...
+        __entries: ClassVar[dict] = ...
+        def __init__(self, value: int) -> None: ...
+        def __eq__(self, other: object) -> bool: ...
+        def __getstate__(self) -> int: ...
+        def __hash__(self) -> int: ...
+        def __index__(self) -> int: ...
+        def __int__(self) -> int: ...
+        def __ne__(self, other: object) -> bool: ...
+        def __setstate__(self, state: int) -> None: ...
+        @property
+        def name(self) -> str: ...
+        @property
+        def value(self) -> int: ...
+
+    class TYPES:
+        __members__: ClassVar[dict] = ...  # read-only
+        ARRAY: ClassVar[Type.TYPES] = ...
+        CLASS: ClassVar[Type.TYPES] = ...
+        PRIMITIVE: ClassVar[Type.TYPES] = ...
+        UNKNOWN: ClassVar[Type.TYPES] = ...
+        __entries: ClassVar[dict] = ...
+        def __init__(self, value: int) -> None: ...
+        def __eq__(self, other: object) -> bool: ...
+        def __getstate__(self) -> int: ...
+        def __hash__(self) -> int: ...
+        def __index__(self) -> int: ...
+        def __int__(self) -> int: ...
+        def __ne__(self, other: object) -> bool: ...
+        def __setstate__(self, state: int) -> None: ...
+        @property
+        def name(self) -> str: ...
+        @property
+        def value(self) -> int: ...
+    def __init__(self, *args, **kwargs) -> None: ...
+    def pretty_name(self, *args, **kwargs) -> Any: ...
+    def __eq__(self, arg0: Type) -> bool: ...
+    def __hash__(self) -> int: ...
+    def __ne__(self, arg0: Type) -> bool: ...
+    @property
+    def dim(self) -> int: ...
+    @property
+    def type(self) -> Type.TYPES: ...
+    @property
+    def underlying_array_type(self) -> Type: ...
+    @property
+    def value(self) -> object: ...
+
+class lief.DEX.MapList.it_items_t:
```

Comment:
Same problem here. `lief.DEX.MapList.it_items_t` is not a valid Python identifier/class name. Those can't have dots in them.